ingest-app/src/cmr/ingest/services/ingest_service.clj

```diff
@@ -397,6 +397,22 @@
           (dissoc :revision-date :transaction-id)
           (update-in [:revision-id] inc)))))
 
+(defn delete-variable
+  "Deletes a tag with the given concept id"
+  [context variable-key]
+  (let [existing-concept (fetch-variable-concept context variable-key)]
+    (mdb/save-concept
+      context
+      (-> existing-concept
+          ;; Remove fields not allowed when creating a tombstone.
+          (dissoc :metadata :format :provider-id :native-id :transaction-id)
+          (assoc :deleted true
+                 :user-id (context-util/context->user-id
+                           context
+                           msg/token-required-for-variable-modification))
+          (dissoc :revision-date :created-at :extra-fields)
```

Comment:
@oubiwann I don't know off the top of my head. I will have to debug it to find out.